waspc/cli/src/Wasp/Cli/Common.hs

```diff
@@ -24,12 +25,17 @@ import qualified Wasp.Util.Terminal as Term
 
 data DotWaspDir -- Here we put everything that wasp generates.
 
+data NodeModulesDir
```

Comment:
What about the `DotWaspDir` defined above?

Reply:
Yeah not sure. At that point there was no Wasp/Project yet.

I am quite sure that NodeMOdulesDir should go into Wasp/Project, because Wasp/Project is all about user's project, and NodeModulesDir is part of it.

DotWaspDir is part of generated stuff, so that is a bit different. I would have to thoroughly inspect Wasp.Project and usage of DotWaspDir in Cli to make this decision. For now I just recommend moving NodeModulesDir to Wasp.Project.... if that makes sense to you (if not, then don't), and we will handle DotWaspDir later (unless it is clear to you where it should go).

Reply:
Whoever ends up implementing this can decide as far as I'm concerned. If you're not happy with a decision, leave a todo in the code and resolve the comment.

Reply:
Moved it into Wasp.Project.